lib/client.ts

```diff
@@ -79,7 +79,12 @@ class ClientManager extends EventTarget {
       password,
     })
     const config = await this._getConfig()
-    const serverConfig = await this._getServerConfig(agent, config.did)
+    // If the labeler needs further configuration, let's generate a mock serverConfig
+    // and let the user move forward with configuration steps
+    const needsConfiguration = config.needs.key || config.needs.service
+    const serverConfig = await (needsConfiguration
+      ? parseServerConfig({})
+      : this._getServerConfig(agent, config.did))
```

Comment:
I'm not sure I understand how this part works.  Ideally IMO the server config and the labeler config could be managed independently from each other, and we wouldn't need to avoid loading the real server config.  But maybe there's an overlap that I'm overlooking.

Reply:
requests to the server config seems to fail until the labeler config is available. the error was something like `record not found` or something along that line and since it doesn't make a lot of sense to fetch this until the labeler config is available, delaying the fetch makes sense to me.

Reply:
(And if we're not sure whether we interpreted the "record not found" error properly, perhaps there's still an issue out there!)

Reply:
Ohhhhkay, I get what's going on now.  They are tangled together a little bit because we need to use service proxying to fetch the server config.  And server proxying is not going to work until the service is reflected in the labeler's DID document.  So we need to wait until `config.needs.service` is `false` before we can make authed requests to ozone.

I made some small tweaks in https://github.com/bluesky-social/ozone/pull/156/commits/fc7093cb460d98ba09518873a91d9788123fc919, let me know what you think!